src/experimental/information_estimates.cpp

```diff
@@ -0,0 +1,568 @@
+#include <math.h>
+#include "slam_toolbox/experimental/information_estimates.hpp"
+
+InformationEstimates::InformationEstimates(kt_double sensor_range, kt_double resolution, kt_double lambda, kt_double nu)
+{
+    m_max_sensor_range = sensor_range;
+    m_cell_resol = resolution;
+    m_obs_lambda = lambda;
+    m_obs_nu = nu;
+}
+
+InformationEstimates::InformationEstimates()
+{
+    m_max_sensor_range = 25.0;
+    m_cell_resol = 0.1;
+    m_obs_lambda = 0.35;
+    m_obs_nu = 0.28;
+}
+
+
+kt_double InformationEstimates::findMutualInfo(std::vector<karto::LocalizedRangeScan*> const& range_scans)
```

Comment:
@hidmic This is the new function.

It is under testing now, but I would like to request a review on this (You are better than me finding errors). I guess this function contains all the functionality now.

PS: Sorry for the Spanish/English comments, they will be removed.